src/amo/css/OverallRating.scss

```diff
@@ -0,0 +1,40 @@
+.OverallRating {
+
+  form {
+    display: flex;
+    justify-content: center;
+
+    input {
+      appearance: none;
+      opacity: 0;
+      width: 0;
+      height: 0;
+    }
+
+    label {
```

Comment:
The browser applies styles from right to left. So any time it encounters a `<label>` it will need to start walking up the tree to figure out if it has a parent `<form>` and that form has a parent `.OverallRating`. With `.OverallRating-label` it just looks at the class and is done. Mostly the issue would be with elements that don't match, where it would need to look all the way up to `<body>` to see that there's no `.OverallRating`.

I don't think there's any sort of optimization to limit based on the class at the left of the selector.